phase1/master.cfg

```diff
@@ -1312,6 +1312,27 @@ def prepareFactory(target):
         )
     )
 
+    factory.addStep(
+        ShellCommand(
+            name="kmodclean",
+            description="Removing kmods from target packages",
+            descriptionDone="Kmods removed",
+            command=[
+                "find",
+                "-name"
```

Comment:
You forgot a „,“ and Python is kind enough do glue two strings together. Right now this breaks the staging buildbot